server/calls.go

```diff
@@ -171,11 +171,14 @@ func initVideoCalls(jsconfig json.RawMessage) error {
 	return nil
 }
 
-func (call *videoCall) messageHead(newState string, duration int) map[string]interface{} {
-	head := map[string]interface{}{
-		"replace": ":" + strconv.Itoa(call.seq),
-		"webrtc":  newState,
+func (call *videoCall) messageHead(head map[string]interface{}, newState string, duration int) map[string]interface{} {
+	if head == nil {
+		head = map[string]interface{}{}
```

Comment:
Any particular need for this?

Reply:
If a call was initiated by a root user on behalf of an ordinary user, the `sender` header should be preserved.

Reply:
I see. I think it's worth leaving a comment here.

Reply:
Comment added.